src/Components/TabletApp/RightSide/RightSide.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Box } from "@mui/system";
+import { Typography } from "@mui/material";
+
+const availabilityMessages = [
+    "currently available",
+    "going to be in use soon",
+    "being used",
+];
+
+const RightSide = ({
```

Comment:
Create interface

Reply:
interfeace iRightSide:{
roomName:string,
availability:number;
}
{...}:iRightSide

Reply:
The interfaces have been added.